pkg/infrastructure/gcp/clusterapi/operation.go

```diff
@@ -11,7 +11,7 @@ import (
 // WaitForOperationGlobal will attempt to wait for a operation to complete where the operational
 // resource is globally scoped.
 func WaitForOperationGlobal(ctx context.Context, projectID string, operation *compute.Operation) error {
-	ctx, cancel := context.WithTimeout(ctx, time.Minute*1)
+	ctx, cancel := context.WithTimeout(ctx, time.Minute*2)
```

Comment:
Can we use this opportunity to fix the error string below at ln 24?

`return fmt.Errorf("failed to wait for global operation: %w", err)`